### PR TITLE
Adapted and extended for THZ404

### PR DESCRIPTION
**Pull request by sg-media-dev**

### CODE REVIEW

File: src/property.h [hunk 1]
```diff
@@ -110,15 +110,17 @@ struct Property : public detail::Property {
...
+    PROPERTY(AKTIVE_HEIZSTUFEN, 0x05bb); //THZ404
     PROPERTY(KUEHLSYSTEM, 0x0613);
     PROPERTY(DRUCK_HEIZKREIS, 0x064a, Type::et_dec_val);
     PROPERTY(LEISTUNG_AUSLEGUNG_KUEHLEN, 0x0692);
     PROPERTY(MOTORSTROM, 0x069f);
     PROPERTY(MOTORLEISTUNG, 0x06a0, Type::et_cent_val);
     PROPERTY(MOTORSPANNUNG, 0x06a1);
     PROPERTY(ABLUFTTEMP, 0x0694, Type::et_dec_val);
-    PROPERTY(ANZEIGE_NIEDERDRUCK, 0x07a7, Type::et_dec_val);
+    PROPERTY(ANZEIGE_HOCHDRUCK, 0x07a6, Type::et_cent_val); //NEU
```
Comment (kr0ner) [inline src/property.h:122]: If you add new values to property.h please guard it with the corresponding flag for your heatpump, no additional comments are needed
```
#ifdef THZ404
PROPERTY(ANZEIGE_HOCHDRUCK, 0x07a6, Type::et_cent_val);
#endif
```
others with different heatpumps can then try it int heir setup and add their flag to enable it ... once all variants confirmed that it works we can set it as default and remove the flags

File: yaml/wp_base.yaml [hunk 2]
```diff
@@ -0,0 +1,640 @@
...
+  MONAT:                          !include { file: wp_generic.yaml, vars: { property: "MONAT"                    , icon: "mdi:calendar" }}
+  WOCHENTAG:                      !include { file: wp_generic.yaml, vars: { property: "WOCHENTAG"                , icon: "mdi:calendar-today" }}
+  TAG:                            !include { file: wp_generic.yaml, vars: { property: "TAG"                      , icon: "mdi:calendar-today" }}
+  STUNDE:                         !include { file: wp_generic.yaml, vars: { property: "STUNDE"                   , icon: "mdi:calendar-clock" }}
+  MINUTE:                         !include { file: wp_generic.yaml, vars: { property: "MINUTE"                   , icon: "mdi:calendar-clock" }}
+  ANZEIGE_NIEDERDRUCK:            !include { file: wp_generic.yaml, vars: { property: "ANZEIGE_NIEDERDRUCK"      , accuracy_decimals: "2" }}
+  ANZEIGE_HOCHDRUCK:              !include { file: wp_generic.yaml, vars: { property: "ANZEIGE_HOCHDRUCK"      , accuracy_decimals: "2" }}
+  #PUMPENZYKLEN_MIN_AUSSENT:       !include { file: wp_generic.yaml, vars: { property: "PUMPENZYKLEN_MIN_AUSSENT" }} # THZ404 -> AKTIVE_HEIZSTUFEN
```
Comment (kr0ner) [inline yaml/wp_base.yaml:300]: The whole idea behind the variant handling is that we no longer have to comment out stuff in the yaml to make it work for a specific heat pump ... the only place where we decide which variant to build is in the esp32-poe-technik.yaml
Those specific packages should first go to thz404.yaml ... others can pick it up and test for e.g. thz504. Once proven that it works for all variants we move it back to this file

**Review by kr0ner — CHANGES_REQUESTED** [review]
Please see the comments ;)

File: src/property.h (hunk 1 above)
Comment (sg-media-dev) [reply src/property.h:122]: Great, the example helps me and I'll test it later :-)

File: yaml/wp_base.yaml (hunk 2 above)
Comment (sg-media-dev) [reply yaml/wp_base.yaml:300]: I will also change it here later ;-)

**Comment by kr0ner** [comment]
closing this because it became outdated

**Pull request closed without merging**